### PR TITLE
Persistent image dev

### PR DESCRIPTION
**Pull request by dsding2**
Change the default behavior for no image url detected to instead pull the last image sent back to the user.

### CODE REVIEW

File: image_utils.py [hunk 1]
```diff
@@ -31,23 +44,41 @@ def spoof_human():
     kwargs: the keyword arguments needed to run func
 """
 async def process_command(ctx, func, *args, **kwargs):
-    if len(args) == 0:
+    image_attached = 0
```
Comment (BrendanParmer) [inline image_utils.py:47]: Probably should be using enums for this to make it more readable

File: main.py [hunk 2]
```diff
@@ -21,11 +21,11 @@ async def on_ready():
...
 
-@bot.command(name="test_image", description="Test that the bot can download images and send them back")
-async def test_image(ctx, *args):
+@bot.command(name="echo", description="Echos back the image")
+async def echo(ctx, *args):
     await process_command(ctx, test_image_fun, *args)
 
-@test_image.error
+@echo.error
 async def image_error_handler(ctx, error):
```
Comment (BrendanParmer) [inline main.py:29]: would do `echo_error_handler` for consistency. 
Maybe we finally move these functions to a different "test" module or something?

**Review by BrendanParmer — CHANGES_REQUESTED** [review]
Performs well and looks good! Just a couple stylistic things for readability/organization

File: main.py (hunk 2 above)
Comment (dsding2) [reply main.py]: I don;t think we need it yet, since we only have this one test function currently.

**Review by BrendanParmer — APPROVED** [review]

**Pull request merged**